mne/datasets/limo/limo.py

```diff
@@ -0,0 +1,253 @@
+# Authors: Jose C. Garcia Alanis <alanis.jcg@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os
+from os import path as op
+
+import re
+import zipfile
+import tarfile
+from sys import stdout
+import scipy.io
+import numpy as np
+import pandas as pd
+import mne
```

Comment:
We typically don't import mne. We just import, as you do below, from relative paths.